packages/toolkit/src/view/dashboard/activity/ModelsTriggerCountsLineChart.tsx

```diff
@@ -30,12 +29,11 @@ function selectGraph(params: any, myChart: echarts.ECharts): void {
 }
 
 /* eslint-disable-next-line  @typescript-eslint/no-explicit-any */
-function unselectGraph(params: any, myChart: echarts.ECharts): void {
```

Comment:
Could you import only the type and leave the typing in place?
like `import type {.ECharts } from "echarts"`

Reply:
addressed in [commit](https://github.com/instill-ai/console/pull/1814/commits/044cd328fea1c197d2bbf05b64162a6849c3eb39).